cfgov/v1/models/base.py

```diff
@@ -58,15 +53,6 @@ def shared_q(self):
     def shared(self):
```

Comment:
Interesting, I'm curious what made it a rabbit hole. These methods don't seem to be used anywhere. I don't think we need a custom `PageQuerySet` anymore without them.